### PR TITLE
Added OpenConnect VPN SSL server role

### PR DESCRIPTION
**Pull request by andreysabitov**
This playbook installs OpenConnect VPN SSL server role. 

To run playbook just execute ```ansible-playbook -i inventory/1-datacenter vpn-server.yml```.
The role will be installed on the **host-02** by default. You can change this in **inventory/1-datacenter** file or **vpn-server.yml**.
Make sure you changed the default **username:password** pair in the **openconnect_users** variable before deploying this role to the server.

The detailed info about the role is in the [README](https://github.com/andreysabitov/microservices-infrastructure/blob/feature/openconnect/roles/openconnect/README.rst) file.

### CODE REVIEW

File: roles/openconnect/README.rst [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+============================  =============================  ============================
+Variable                      Description                    Default value
+============================  =============================  ============================
+openconnect_int_net           tunnel network                 192.168.253.0
+openconnect_int_mask          mask for the tunnel network    255.255.255.0
+openconnect_dns               DNS server for a client        8.8.8.8
+openconnect_route_list        list with injected routes      directly connected network
+============================  =============================  ============================
+
+By default OpenConnect server inject route for directly connected networki to the client.
```
Comment (OleksandrBerchenko) [inline roles/openconnect/README.rst]: network

File: vpn-server.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+- hosts: dc1:&vpn_servers
```
Comment (OleksandrBerchenko) [inline vpn-server.yml]: dc1 only?

Comment (andreysabitov) [inline vpn-server.yml]: Ok, I'll add the same group into dc2 inventory. User will be able to choose necessary servers with the inventory parameter.

**Comment by OleksandrBerchenko** [comment]
Looks good to me.

Ready for testing (if it's not tested yet).

By the way, I'm not able to merge it automatically.

**Comment by OleksandrBerchenko** [comment]
And please add a link to the README to the docs folder (see other roles as a reference).

**Pull request closed without merging**